src/modules/Telemetry/Sensor/BMP3XXSensor.cpp

```diff
@@ -0,0 +1,62 @@
+#include "configuration.h"
+
+#if !MESHTASTIC_EXCLUDE_ENVIRONMENTAL_SENSOR
+
+#include "../mesh/generated/meshtastic/telemetry.pb.h"
+#include "BMP3XXSensor.h"
+#include "TelemetrySensor.h"
+#include <Adafruit_BMP3XX.h>
+#include <typeinfo>
+
+BMP3XXSensor::BMP3XXSensor() : TelemetrySensor(meshtastic_TelemetrySensorType_BMP3XX, "BMP3XX"){}
+
+BMP3XXSensor bmp3xxSensor;
+
```

Comment:
Please put that in the telemetry init and reference it from there.

Reply:
Ok that's done